### PR TITLE
[bitnami/nginx] Add resources for Add resources to Git-Sync. Otherwise hpa did not work

### PR DESCRIPTION
**Pull request by MathysDev**
I had problems that hpa did not work together with GitClone

Added resources to Sync-Container in deployment.
In my Test-Environment i was able to use hpa and GitClone together

Replaces #16185

### CODE REVIEW

**Comment by migruiz4** [comment]
Hi @MathysDev,

Please notice your PR includes changes in several charts and appears to be checkout from another branch different from `main`.

Could you please fix it?

File: bitnami/nginx/values.yaml [hunk 1]
```diff
@@ -256,6 +256,30 @@ resources:
...
+  ##    memory: 128Mi
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    cpu: 100m
+    ##    memory: 128Mi
+    requests: {}
+
+
+
```
Comment (migruiz4) [inline bitnami/nginx/values.yaml:282]: Could you please remove the extra empty lines please?

File: bitnami/nginx/values.yaml [hunk 2]
```diff
@@ -256,6 +256,30 @@ resources:
...
+## syncer containers' resource requests and limits
+## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+## We usually recommend not to specify default resources and to leave this as a conscious
+## choice for the user. This also increases chances charts run on environments with little
+## resources, such as Minikube. If you do want to specify resources, uncomment the following
+## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+## @param syncer.resources.limits The resources limits for the Syncer-container
+## @param syncer.resources.requests The requested resources for the Syncer-container
+##  
+syncer:
```
Comment (migruiz4) [inline bitnami/nginx/values.yaml:268]: Please move this under the existing object as `cloneStaticSiteFromGit.gitSync.resources`

File: bitnami/nginx/templates/deployment.yaml [hunk 3]
```diff
@@ -133,6 +133,9 @@ spec:
           {{- if .Values.cloneStaticSiteFromGit.gitSync.args }}
           args: {{- include "common.tplvalues.render" (dict "value" .Values.cloneStaticSiteFromGit.gitSync.args "context" $) | nindent 12 }}
           {{- end }}
+          {{- if .Values.syncer.resources }}
+          resources: {{- toYaml .Values.syncer.resources | nindent 12 }}
+          {{- end }}
```
Comment (migruiz4) [inline bitnami/nginx/templates/deployment.yaml:138]: Please rename the values to match the existing settings for the `git-repo-syncer` container
```suggestion
          {{- if .Values.cloneStaticSiteFromGit.gitSync.resources }}
          resources: {{- toYaml .Values.cloneStaticSiteFromGit.gitSync.resources | nindent 12 }}
          {{- end }}
```

File: bitnami/nginx/values.yaml [hunk 4]
```diff
@@ -256,6 +256,30 @@ resources:
...
+  ## Example:
+  ## limits:
+  ##    cpu: 100m
+  ##    memory: 128Mi
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    cpu: 100m
+    ##    memory: 128Mi
+    requests: {}
```
Comment (migruiz4) [inline bitnami/nginx/values.yaml:279]: As it is a secondary setting, we can keep the description shorter here:

```suggestion
  ## git-repo-syncer resource requests and limits
  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
  ## @param cloneStaticSiteFromGit.gitSync.resources.limits The resources limits for the git-repo-syncer container
  ## @param cloneStaticSiteFromGit.gitSync.resources.requests The requested resources for the git-repo-syncer container
  ##
  resources:
    limits: {}
    requests: {}
```

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Thank you for fixing the branch issues! Please take a look at my suggestions.

File: bitnami/nginx/values.yaml (hunk 1 above)
Comment (MathysDev) [reply bitnami/nginx/values.yaml]: removed with https://github.com/bitnami/charts/pull/16204/commits/0bfa951e4a749d1d18dc64e3613221f6818d299b

File: bitnami/nginx/values.yaml (hunk 2 above)
Comment (MathysDev) [reply bitnami/nginx/values.yaml]: Done in https://github.com/bitnami/charts/pull/16204/commits/528fec25178785e79daa695b55d496d7f292943c

**Comment by MathysDev** [comment]
@migruiz4 Thank you very much for your feedback!

**Review by migruiz4 — APPROVED** [review]
Looks good to me! Thank you for applying the suggested changes!

**Pull request merged**